Libraries/Image/ImageStylePropTypes.js

```diff
@@ -32,6 +32,12 @@ var ImageStylePropTypes = {
   // It changes the color of all the non-transparent pixels to the tintColor
   tintColor: ColorPropType,
   opacity: ReactPropTypes.number,
+
+  // Android-Specific styles
```

Comment:
Image (on iOS, at least) only supportsa subset of the border styles. It doesn't support individual corner radii or different border colors or widths for each side.

For that reason, having the styles specified separately for Image is intentional.

This diff adds additional border styles for Image on Android that aren't supported on iOS. I'm not sure what the best way to handle that is.

Maybe we could just implement these styles for View for now and not Image? I don't think there's a lot of demand for individually specified corners radii on images.